### PR TITLE
cleanup leftovers

### LINKED ISSUE #18908: Clean up leftovers from prs
more details: https://github.com/status-im/status-mobile/pull/18907/files#r1494852579

### PR DESCRIPTION
**Pull request by Parveshdhull**
fixes https://github.com/status-im/status-mobile/issues/18908

### Summary

PR addresses some leftovers

### Testing
PR doesn't to be manually tested.

status: ready

### CODE REVIEW

File: src/legacy/status_im/multiaccounts/logout/core.cljs [hunk 1]
```diff
@@ -31,7 +31,6 @@
   (let [key-uid (get-in db [:profile/profile :key-uid])]
     (rf/merge cofx
               {:set-root                               :progress
-               :chat.ui/clear-inputs                   nil
```
Comment (Parveshdhull) [inline src/legacy/status_im/multiaccounts/logout/core.cljs:34]: No effect `:chat.ui/clear-inputs`

File: src/quo/components/navigation/bottom_nav_tab/view.cljs [hunk 2]
```diff
@@ -32,7 +32,7 @@
   "
   [{:keys [icon new-notifications? notification-indicator counter-label
            on-press pass-through? icon-color-anim accessibility-label test-ID
-           customization-color on-long-press]
```
Comment (Parveshdhull) [inline src/quo/components/navigation/bottom_nav_tab/view.cljs:35]: leftover from https://github.com/status-im/status-mobile/pull/18165

File: src/status_im/contexts/communities/events.cljs [hunk 3]
```diff
@@ -229,10 +229,9 @@
     {:db (update-in db
                     [:communities community-id]
                     (fn [community]
-                      (-> community
```
Comment (Parveshdhull) [inline src/status_im/contexts/communities/events.cljs]: leftover from https://github.com/status-im/status-mobile/pull/18843#discussion_r1492227751

File: src/status_im/contexts/shell/jump_to/view.cljs [hunk 4]
```diff
@@ -15,12 +15,10 @@
 
 (defn navigate-back-handler
   []
-  (if (and (not @navigation.state/curr-modal)
-           (seq (utils/open-floating-screens)))
-    (do
-      (rf/dispatch [:navigate-back])
-      true)
-    false))
```
Comment (Parveshdhull) [inline src/status_im/contexts/shell/jump_to/view.cljs:23]: no need to return false

File: src/legacy/status_im/multiaccounts/logout/core.cljs (hunk 1 above)
Comment (J-Son89) [reply src/legacy/status_im/multiaccounts/logout/core.cljs:34]: thank you!

File: src/quo/components/navigation/bottom_nav_tab/view.cljs [hunk 5]
```diff
@@ -48,7 +48,6 @@
                                     :border-radius 10})]
     [rn/touchable-without-feedback
      {:test-ID                 test-ID
-      :on-long-press           on-long-press ;;NOTE - this is temporary while supporting old wallet
```
Comment (J-Son89) [inline src/quo/components/navigation/bottom_nav_tab/view.cljs:51]: 🙌

**Review by J-Son89 — APPROVED** [review]

**Review by ajayesivan — APPROVED** [review]
Thank you for the PR!

File: src/status_im/contexts/communities/events.cljs (hunk 3 above)
Comment (Parveshdhull) [reply src/status_im/contexts/communities/events.cljs]: reverted, addressed in https://github.com/status-im/status-mobile/pull/18870

**Pull request merged**